src/api/error.ts

```
@@ -17,29 +17,28 @@ export class KiwiError extends Error {
 
   constructor(options: KiwiErrorOptions) {
     super(options.message);
-    this.name = 'KiwiError';
+    this.name = "KiwiError";
     this.level = options.level || Levels.ERROR;
     this.cause = options.cause;
   }
 }
 
 export class ThreadErrorHandler {
```

ESLint

[error] Unexpected class with only static properties.

(`@typescript-eslint/no-extraneous-class`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected console statement. Only these console methods are allowed: debug.

(no-console)